src/Components/Lodging/Lodging.js

```diff
@@ -0,0 +1,192 @@
+import React, { Component } from 'react';
+import { 
+  StyleSheet, 
+  Text, 
+  View, 
+  ScrollView,
+  TouchableOpacity 
+} from 'react-native';
+import WandererFooter from '../WandererFooter/WandererFooter';
+import WandererHeader from '../WandererHeader/WandererHeader';
+import { withNavigationFocus } from 'react-navigation';
+import { fetchLodging } from '../../util/apiCalls'
+
+
+class Lodging extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      leg: this.props.navigation.getParam('leg'),
+      lodgings: [],
+      error: '',
+      existingLegId: this.props.navigation.getParam('existingLegId') || null
+     }
+  }
+
+  componentDidMount = async () => {
+    try {
+      let lodgings = await fetchLodging(this.state.leg.id)
+      this.setState({lodgings})
+    } catch (error) {
+      this.setState({error: 'There was an error fetching your lodging'})
+    }
+  }
+
+  componentDidUpdate = async (prevProps) => {
+    if (prevProps.isFocused !== this.props.isFocused) {
+      this.componentDidMount();
+    }
+  }
+
+  displayLodging = () => {
+    const {navigate} = this.props.navigation;
+    return this.state.lodgings.map((lodging, index) => {
+      return (
+        <View style={styles.borderContainer}>
+          <Text style={styles.headerText}>{lodging.name} details</Text>
+          <Text style={styles.text}>City {lodging.city}</Text>
+          <Text style={styles.text}>Arriving {lodging.arrivalDate}</Text>
+          <Text style={styles.text}>Leaving {lodging.departureDate}</Text>
+          <TouchableOpacity key={index + lodging.id} style={styles.tripButton}>
+            <Text onPress={() => navigate('AddLodgingInfo', {leg: this.state.leg, legId: this.state.leg.id, lodgingId: lodging.id, lodging: lodging})} style={styles.text} key={lodging.id}>Edit Lodging</Text>
+          </TouchableOpacity>
+        </View>
+    )
+      })
+  }
+
+  render() {
+  const { leg, error, lodgings } = this.state
+  const {navigate} = this.props.navigation;
+  return (
+      <View style={styles.container}>
+
+      <WandererHeader />
+
+        <ScrollView>
+          <Text style={styles.title}>Leg Lodging Detail</Text>
+          <Text style={styles.text}>Leg: {leg.startLocation} - {leg.endLocation}</Text>
+
+          <TouchableOpacity>
+            <Text onPress={() => navigate('AddLodgingInfo', {legId: leg.id})} style={styles.button}>Add Lodging</Text>
```

Comment:
@EmilyLalonde I think that we need to pass a userId through to the AddLodgingInfo component. You can test by navigating to the AddLodgingInfo page and then clicking on Followers.